redfish-core/lib/log_services.hpp

```diff
@@ -4993,15 +4993,18 @@ inline void fillSystemHardwareIsolationLogEntry(
                     {
                         entryJson["Severity"] = "Critical";
                     }
-                    else if ((*severity == "xyz.openbmc_project."
-                                           "HardwareIsolation."
-                                           "Entry.Type.Warning") ||
-                             (*severity == "xyz.openbmc_project."
-                                           "HardwareIsolation."
-                                           "Entry.Type.Manual"))
+                    else if (*severity == "xyz.openbmc_project."
+                                          "HardwareIsolation."
+                                          "Entry.Type.Warning")
                     {
                         entryJson["Severity"] = "Warning";
                     }
+                    else if (*severity == "xyz.openbmc_project."
+                                          "HardwareIsolation."
+                                          "Entry.Type.Manual")
+                    {
+                        entryJson["Severity"] = "OK";
```

Comment:
This seems really random. I don't like this, it is unexpected. https://github.com/ibm-openbmc/openpower-hw-isolation/issues/36 is an issue to revisit this